UnityProject/Assets/Scripts/Core/Editor/Addressable editor/AddressablesInEditor.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+using UnityEngine.AddressableAssets.ResourceLocators;
+using UnityEngine.ResourceManagement.AsyncOperations;
+using UnityEngine.ResourceManagement.ResourceLocations;
+
+
+public class AddressablesInEditor : EditorWindow
+{
+	public static List<IResourceLocation> locations = new List<IResourceLocation>();
+
+	// [MenuItem("Tools/LoadCatalog")]
+	// public static void LoadCatalog()
```

Comment:
Looks like this main menu method is commented out, is this another test file that could be deleted?

Reply:
yeah